Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM python:alpine
+FROM python:3-alpine
 
 WORKDIR /usr/src/app
 
-RUN apk add gcc musl-dev libffi-dev openssl-dev libxml2-dev libxslt-dev file llvm-dev make g++
-
 COPY requirements.txt ./
 
-RUN pip install wheel cryptography gunicorn pymysql
-RUN pip install --no-cache-dir -r requirements.txt
+RUN apk --no-cache add gcc musl-dev libffi-dev openssl-dev libxml2-dev libxslt-dev file llvm-dev make g++ \
```

Comment:
You should look into ``--virtual`` to not duplicate packages and forget some.

This should be a good ref https://stackoverflow.com/questions/46221063/what-is-build-deps-for-apk-add-virtual-command/46222036

Reply:
What do you think would be best way to do this? `libxml2-dev` and `libxslt-dev` are required for runtime

Reply:
``lixml2`` and ``lixslt`` are required at runtime. I normally install the dev variant as a virtual package and the normal one not as a virtual. So only the virtuals are removed at the end.